js/main.js

```diff
@@ -1,2 +1,9 @@
 /* global data */
 /* exported data */
+var $img = document.querySelector('img');
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function (event) {
+  $form.reset();
+  $img.src = 'images/placeholder-image-square.jpg';
+});
```

Comment:
this should  be in the same callback function as the form data to combine the functionality.

Reply:
Thanks @Zhiwutian , I've made those changes as suggested and updated the pull request